stable/redis-ha/templates/redis-pvc.yaml

```diff
@@ -3,7 +3,7 @@
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: {{ template "fullname" . }}-slave
+  name: {{ template "fullname" . }}-pvc
```

Comment:
Since the type is already know to be a PVC it is more indicative to leave a more descriptive name of what the PVC is for.

Reply:
Donezo.